tools/cockpit.spec

```diff
@@ -499,6 +514,34 @@ test -f %{_bindir}/firewall-cmd && firewall-cmd --reload --quiet || true
 %postun ws
 %systemd_postun_with_restart cockpit.socket cockpit.service
 
+%package selinux-policy
+Summary: SELinux policies
+Requires: cockpit-ws = %{version}-%{release}
```

Comment:
If you decide to stick with cockpit-selinux-policy, it would be best to reverse this dependency (cockpit-ws requires cockpit-selinux-policy) so that the policy is installed first. Also, it could cause circular dependency in the future.